server/@types/journeys/index.d.ts

```diff
@@ -41,4 +42,6 @@ declare namespace journeys {
 
   type YesOrNo = 'YES' | 'NO'
   type YesNoOrDoNotKnow = 'YES' | 'NO' | 'DO_NOT_KNOW'
+
+  type PrisonerJourneyParams = { prisonerNumber: string; journeyId: string }
 }
```

Comment:
Interesting. I was thinking yesterday that we'd need to store the `journeyId` from the previous journey if we switch between `manage` and `create` .. I see you've had the same thought!

Reply:
Well this is more for the path params as I was copying `{ prisonerNumber: string; journeyId: string }` into dozens of controller methods but yes, we probably need to share the journey id.